### PR TITLE
input form page and html implemented

### PR DESCRIPTION
**Pull request by CarlOllington**

### CODE REVIEW

File: witw_styles.css [hunk 1]
```diff
@@ -24,15 +24,16 @@ Midnight Shadow (Blue): #555e6e.
 .footer
 {
     display: flex;
-    justify-content: space-between;
+    justify-content: space-evenly;
     color: #6d6e6c;
+    margin-top: auto;
 }
 .btt
```
Comment (Henry-Tottle) [inline witw_styles.css:32]: line space

File: witw_styles.css [hunk 2]
```diff
@@ -111,4 +112,37 @@ Midnight Shadow (Blue): #555e6e.
...
+
+.page_input_form
+{
+    background-color: #171b1b;
+    color: whitesmoke;
+    font-family: "Roboto Thin", sans-serif;
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
```
Comment (Henry-Tottle) [inline witw_styles.css:130]: Line space

File: input_form.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+<head>
+    <title>New Shirt Form</title>
+    <meta charset="UTF-8" name="viewport" content="width=device-width,initial-scale=1" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="witw_styles.css" rel="stylesheet">
+</head>
+
+<body class="page_input_form">
+
+<header class="header">
```
Comment (Henry-Tottle) [inline input_form.php]: Do you need the class name, could target the header tag?  But I'm just nitpicking, and might be wrong

**Review by Henry-Tottle — COMMENTED** [review]
Looks all good, just some things to consider, although I don't have the confidence yet to confirm whether what I've suggested is right

File: witw_styles.css (hunk 1 above)
Comment (CarlOllington) [reply witw_styles.css:32]: implemented.

File: input_form.php (hunk 3 above)
Comment (CarlOllington) [reply input_form.php]: Good call. implemented.

File: witw_styles.css (hunk 2 above)
Comment (CarlOllington) [reply witw_styles.css:130]: implemented.

**Pull request merged**